template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/hbase/HbaseCloudStorageServiceConfigProvider.java

```diff
@@ -60,7 +61,7 @@ public boolean isConfigurationNeeded(CmTemplateProcessor cmTemplateProcessor, Te
                 entitlementService.sdxHbaseCloudStorageEnabled(accountId);
```

Comment:
Nit: Please join the expression with the preceding line.

Reply:
done